src/components/AddFoodForm.jsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from 'react';
+import { Row, Col } from 'antd';
+
+//Create Form
+function AddFood(props) {
+  const {createFood} = props
+  const [name, setName] = useState('');
+  const [image, setImage] = useState('');
+  const [calories, setCalories] = useState(0);
+  const [servings, setServings] = useState(0);
+
+  const handleName = (e) => setName(e.target.value);
+  const handleImage = (e) => setImage(e.target.value);
+  const handleCalories = (e) => setCalories(e.target.checked);
```

Comment:
When you create a new food the calories are not being updated because here you have `e.target.checked` instead of `e.target.value` change the checked to value and it will work!